packages/origin-backend/src/pods/admin/admin.service.ts

```diff
@@ -19,6 +19,17 @@ export class AdminService {
         });
     }
 
+    public async getUsersBy(orgName: string, status: number, kycStatus: number) {
+        const result = await this.repository.find({
+            where: { status, kycStatus },
+            relations: ['organization']
+        });
+        return result.filter((item) => {
```

Comment:
Read this on how to handle case insensitive queries https://github.com/typeorm/typeorm/issues/1231

Reply:
have do something as below , but not working . any other sample that can help on join

public async getUsersBy(orgName: string, status: Status, kycStatus: KYCStatus) {
        console.log({ name:Like(`%${orgName}%`) });
        const result = await this.repository.find({
            where: { status, kycStatus, organization:{ name: Raw(name => `LOWER(${name}) Like '%${orgName.toLowerCase()}%`)}  },
            relations: ['organization']
        });
        console.log(result);
        return result;